anet-dictionary.yml

```diff
@@ -321,18 +321,34 @@ dictionary:
               typeError: Qty must be a number
               label: Qty
           visibleWhen: $[?(@.multipleButtons && (@.multipleButtons.indexOf('assist') != -1 || @.multipleButtons.indexOf('other') != -1))]
-
+    # cF_ => lifted custom fields
     person:
-      firstName: First name
-      lastName: Last name
-      domainUsername: Domain username
+      numberOfFieldsUnderAvatar: 7
+      firstName:
+        label: First name
+        inShowPage: false
```

Comment:
Wouldn't it be easier to just specify a list of fields show on the Show page, e.g.:
```
      fieldsOnShowPage:
      - domainUsername
      - emailAddress
      - …etc…
```
and leave the rest unchanged?

Reply:
My first thinking was doing like this, then i thought, this would duplicate the field names, needs to be synced whenever something is added etc.

Reply:
From my point of view, having these together in one list is clearer and more convenient.